lib/pool.py

```diff
@@ -1,14 +1,15 @@
 import logging
 import os
 import traceback
-from typing import Any, Dict, Optional, cast
+from typing import Any, Dict, Optional
 
 from packaging import version
 
 import lib.commands as commands
 from lib.common import _param_get, _param_set, safe_split, wait_for, wait_for_not
 from lib.host import Host
 from lib.sr import SR
+import contextlib
```

Comment:
Is it the right place for this import? I'd expect it grouped with system imports.

Reply:
yes, it also shows that there should be a rule for that :)